app/src/main/res/values/colors.xml

```diff
@@ -113,4 +113,19 @@
 
     <color name="settings_summary_text">#ffbbbcc5</color>
     <color name="settings_whats_new">#FFFF1AD9</color>
+
+    <!-- Dark theme -->
+    <color name="menu_dark_theme">#342F6D</color>
```

Comment:
Let's take our learnings from Fenix and avoid some of the mistakes there, with the context Focus.

Fenix, had three themes: light, dark, private. The system did not support the last one because it was more of a "mode" than a theme, so what our styles/color files looked like were light, dark, light-private, dark-private.

Focus, does not have this complexity. We only have light and dark, so we can use the system to manage these for us.

What this means is that we can use the `values-night` directory to hold our dark theme based colours/themes and our `values` for the light theme.

Let's no duplicate attributes that are meant only for dark mode in both files.

Reply:
I was going to ask you about this today but you beat me to it 😉

Reply:
> The easiest way to do this is to have them duplicated now and let the system handle the switching. Then, we can come in later and change the values in colors.xml to light theme without touching dark.

I wouldn't call it duplicating, instead the same **reference** colour reference name is available in `values-night/colors` and `values/color` with the color **value** in each file being different.

Let me know if the root comment makes sense or not. I can give a more examples if needed.

Reply:
Yes, it makes sense. I think we are describing the same thing with different words :) You want the reference color in colors.xml (which references the same colors as night/colors.xml for now) instead of doing this: 

```
<color name="headerGradientStart">@color/headerGradientStart_dark_theme</color>
```